lib/baseTest.py

```diff
@@ -132,21 +137,48 @@ def remove_test_file(self):
         if hasattr(self, "test_url_id"):
             self.target_helper.delete_target(self.test_url_id)
 
-    def load_configs(self):
+    def platform_dep_settings_handler(self, config_value):
+        if self.current_platform_name in config_value:
+            if self.current_platform_ver in config_value[self.current_platform_name]:
+                platform_dep_variables = copy.deepcopy(config_value[self.current_platform_name][self.current_platform_ver])
+            else:
+                platform_dep_variables = copy.deepcopy(config_value[self.current_platform_name][''])
```

Comment:
In my opinion, using `""` here as default value is too tricky. (Ref: https://github.com/Mozilla-TWQA/Hasal/pull/626/files#diff-daa66495dbc47f258f4c68bde193e7b3R16)

I'd like to use `"default"` for making code easier to understand.